### PR TITLE
[106X] MTV : adjust impact parameter selection (backport of #26700)

### PR DESCRIPTION
**Pull request by fabiocos**
#### PR description:

See the discussion in https://github.com/cms-sw/cmssw/pull/26700#issue-277019357

#### PR validation:

All tests passed

### CODE REVIEW

**Comment by fabiocos** [comment]
@mtosi FYI

**Comment by fabiocos** [comment]
please test

**Comment by mtosi** [comment]
@hajohajo @jalimena @felicepantaleo @fwyzard

**Comment by jalimena** [comment]
@emiglior @makortel

File: Validation/RecoTrack/python/MultiTrackValidatorGenPs_cfi.py [hunk 1]
```diff
@@ -69,3 +69,8 @@
     doRecoTrackPlots = cms.untracked.bool(True),
     dodEdxPlots = cms.untracked.bool(False),
 )
+
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+premix_stage2.toModify(multiTrackValidator,
+    label_tv = "mixData:MergedTrackTruth",
+)
```
Comment (makortel) [inline Validation/RecoTrack/python/MultiTrackValidatorGenPs_cfi.py]: I missed this change in #26700. This customization is irrelevant as the parameter is not used by `MultiTrackValidatorGenPs` (but it doesn't hurt either).

**Comment by jfernan2** [comment]
@mtosi this PR is only changing histograms in 1XXXX and 2XXXX workflows but 25202, I understand this is the expected behaviour right?
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_6_X_2019-05-14-2300+a701b2/31690/dqm-histo-comparison-summary.html

**Comment by mtosi** [comment]
ciao
thanks for this comment
I've just checked the workflow (by heart I do not remember the map) [1]
the 25202 refers to --era Run2_2016, which has the Phase0 pixel detector
and therefore the original cut on the d_0 is perfectly fine

[1]
runTheMatrix.py -l 25202 --dryRun

**Comment by jfernan2** [comment]
Thank you @mtosi  
FYI, the workflow map is on: 
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_6_X_2019-05-14-2300+a701b2/31690/

**Comment by jfernan2** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**